fe/fe-core/src/main/java/com/starrocks/catalog/CatalogUtils.java

```diff
@@ -309,14 +309,32 @@ public static void checkPartitionValuesExistForAddListPartition(OlapTable olapTa
         }
     }
 
+    public static int divisibleBucketNum(int backendNum) {
```

Comment:
The result number is a value in 1, 2, 3, 4, 5, 6?

Reply:
1-7